.github/workflows/push.yml

```diff
@@ -27,10 +27,10 @@ jobs:
       - name: Checkout ${{ github.ref }}
         uses: actions/checkout@v2
 
-      - name: Setup Node ${{ github.event.inputs.node || github.event.inputs.node.default }}
```

Comment:
How about we define an environment variable like this:
```
env:
  NODE_VERSION: ${{ github.event.inputs.node || github.event.inputs.node.default || 14 }}
```
Then, in all subsequent instances, we simply use `env.NODE_VERSION` which will avoid code duplication and defining the default at multiple locations which is harder to maintain.

Reply:
I think maintenance in this context is really a not an issue (and even lack-there-of will also not really be an issue) and I personally try to avoid setting env vars that act is run time globals. That tends to be what is hard to maintain...

As-changed (14 hard coded), As-pulled (whatever setup/node defaults to) are similar for the workflow task. As-changed is easiest to understand me think. It was a good catch.